pkg/component/observability/monitoring/prometheus/garden/prometheusrules.go

```diff
@@ -68,14 +65,176 @@ func init() {
 }
 
 // CentralPrometheusRules returns the central PrometheusRule resources for the garden prometheus.
-func CentralPrometheusRules() []*monitoringv1.PrometheusRule {
+func CentralPrometheusRules(isGardenerDiscoveryServerEnabled bool) []*monitoringv1.PrometheusRule {
 	return []*monitoringv1.PrometheusRule{
 		auditLog.DeepCopy(),
 		etcd.DeepCopy(),
-		garden.DeepCopy(),
+		gardenPrometheusRule(isGardenerDiscoveryServerEnabled).DeepCopy(),
 		metering.DeepCopy(),
 		recording.DeepCopy(),
 		seed.DeepCopy(),
 		shoot.DeepCopy(),
 	}
 }
+
+func gardenPrometheusRule(isGardenerDiscoveryServerEnabled bool) *monitoringv1.PrometheusRule {
+	getLabels := func(severity string) map[string]string {
+		return map[string]string{
+			"severity": severity,
+			"topology": "garden",
+		}
+	}
+
+	rules := []monitoringv1.Rule{
+		{
+			Alert:  "DashboardDown",
+			Expr:   intstr.FromString("probe_success{job = \"blackbox-dashboard\",purpose = \"availability\"} == 0"),
```

Comment:
Nit: Please use `` instead of "" so that we don't have to escape the quotes in the query (not only here, but also in other `Expr`s and `description`s).

Reply:
I wanted to do it like this, however the description has expressions that are already using the back quote which unfortunately cannot be escaped, it would require some mixed usage of back quote and double quotes like

```go
foo := `hello world
value = ` + "`" + `some_expr{app="bar"}` +"`" + `
lorem ipsum
`
```

And this is already looking confusing and hard to read, therefore I decided to go on with double quotes and escape the one that are in the string. For consistency, the all other string values are using double quotes.

Reply:
I see, fair enough.